packages/fxa-settings/config/webpackDevServer.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = function (proxy, allowedHost) {
       'Access-Control-Allow-Headers': '*',
     },
     // Enable gzip compression of generated files.
-    compress: true,
+    compress: false,
```

Comment:
[See Slack thread](https://mozilla.slack.com/archives/CLV3KMZ8B/p1751385961279919?thread_ts=1751056116.880179&cid=CLV3KMZ8B).

I couldn't modify the public settings file without getting a content encoding error and got it working locally by doing this:
1. Set `compress: false` in this file
`. `pm2 restart settings-react`
1. Load up localhost:3030. THEN, make changes to the public index file. Oddly when I have a change in that file and then restart, I get the same error, but once it loads the first time, it seems OK with changes?

And now that functional tests are failing with the same error, I'm going to see if this fixes it. It's possible this is related to what Amri mentioned in that Slack thread.

Reply:
I was able to get this fix in.